mesures_with_latency_migr.npf

```diff
@@ -7,26 +7,36 @@ n_runs=10
 timeout=150
 
 %variables
+
 DELAY={0,5,10,25,50}
 
+TUNNEL_PENALTY=1
+
+heatmap:DELAY=0
+heatmap:SERVER_DELAY={0,5,10,25,50}
+heatmap:TUNNEL_DELAY={0,5,10,25,50}
+
+%late_variables
```

Comment:
Maybe those two lines deserve an explanation. If SERVER_DELAY and TUNNEL_DELAY are not already defined (so no heatmap tag) then they take the value of DELAY. late_variables is evaluated for each run just before the test so the value of DELAY will be the one instanciated according to the variables for each run